actioncable/lib/action_cable.rb

```diff
@@ -44,7 +44,7 @@ module ActionCable
       server_restart: "server_restart"
     },
     default_mount_path: "/cable",
-    protocols: ["actioncable-v1-json", "actioncable-unsupported"].freeze
+    protocols: %w(actioncable-v1-json actioncable-unsupported).freeze
```

Comment:
This change is not needed. Can you please revert?

Reply:
@rafaelfranca just curious to know why we prefer `[]` instead of `%w()`?

Reply:
It is not a question of preference, it is that the code as it was is fine and we don't accept cosmetic changes in PRs.

Reply:
cool